openassessment/xblock/grade_mixin.py

```diff
@@ -141,6 +142,7 @@ def render_grade_complete(self, workflow):
             'feedback_text': feedback_text,
             'has_submitted_feedback': has_submitted_feedback,
             'code_language': get_code_language(context["student_submission"]['answer']['language']),
+            'design_problem': TestGrader.is_design_problem(self.display_name),
```

Comment:
where is this function defined? `is_design_problem`

Reply:
in code_grader.py file(https://github.com/arbisoft/edx-ora2-1/blob/ce3ae285e59427056492aaa3bb5fc05cf0d7a038/openassessment/xblock/job_sample_grader/code_grader.py#L376)